go/host/client_api_eth.go

```diff
@@ -30,7 +31,8 @@ func (api *EthereumAPI) ChainId() (*hexutil.Big, error) { //nolint:stylecheck,re
 
 // BlockNumber returns the height of the current head rollup.
 func (api *EthereumAPI) BlockNumber() hexutil.Uint64 {
-	return hexutil.Uint64(api.host.nodeDB.GetCurrentRollupHead().Number.Uint64())
+	number := api.host.nodeDB.GetCurrentRollupHead().Header.Number.Uint64()
```

Comment:
No particular reason for this change, just the previous one-liner was a bit big.